kolibri/core/tasks/job.py

```diff
@@ -149,20 +195,8 @@ def __init__(self, func, *args, **kwargs):
         self.args = args
         self.kwargs = kwargs
         self.result = None
-
         self.storage = None
-
-        if callable(func):
```

Comment:
Any reason to remove this code? Seems like harmless defensive programming to me.